src/daemon/DisplayManager.cpp

```diff
@@ -129,11 +129,7 @@ namespace SDDM {
         }
     }
 
-    DisplayManagerSeat::DisplayManagerSeat(const QString &name, QObject *parent) : QObject(parent) {
-        // set name and path
-        m_name = name;
-        m_path = DISPLAYMANAGER_SEAT_PATH + name.mid(4);
-
+    DisplayManagerSeat::DisplayManagerSeat(const QString &name, QObject *parent) : QObject(parent), m_name(name), m_path(DISPLAYMANAGER_SEAT_PATH + name.mid(4)) {
```

Comment:
Newline before the `:` would make the line shorter

Reply:
@Vogtinator I didn't know what coding style to follow.
Will fixit.

Is this good to go ?

``` diff
diff --git a/src/daemon/DisplayManager.cpp b/src/daemon/DisplayManager.cpp
index 531d715..08570dc 100644
--- a/src/daemon/DisplayManager.cpp
+++ b/src/daemon/DisplayManager.cpp
@@ -129,7 +129,8 @@ namespace SDDM {
         }
     }
 
-    DisplayManagerSeat::DisplayManagerSeat(const QString &name, QObject *parent) : QObject(parent), m_name(name), m_path(DISPLAYMANAGER_SEAT_PATH + name.mid(4)) {
+    DisplayManagerSeat::DisplayManagerSeat(const QString &name, QObject *parent) :
+        QObject(parent), m_name(name), m_path(DISPLAYMANAGER_SEAT_PATH + name.mid(4)) {
         // create adaptor
         new SeatAdaptor(this);
```

Reply:
Usually the line break is before the `:`, but I don't mind either way. That line is perfectly readable now.

Reply:
I misread what you have written.